server.go

```diff
@@ -1915,6 +1915,7 @@ func (s *Server) getCodec(contentSubtype string) baseCodec {
 	}
 	codec := encoding.GetCodec(contentSubtype)
 	if codec == nil {
+		logger.Warningf("Unsupported codec %q. Defaulting to %q for now. This will start to fail in future releases.", contentSubtype, proto.Name)
```

Comment:
Is there a similar code path in `handler_server.go` that we also need to change?

Reply:
Both the handler_server and the regular grpc server read the content-subtype from the incoming request before creating the stream, and set the `contentSubType` field in the stream type. So, once we are here, the two paths have already converged.